### DISCUSSION TITLE
Possible issue with formula in Vignettes file

### DISCUSSION THREAD

**Discussion by xjtusjtu** [bmcclintock/momentuHMM#171] [Q&A] [answered]
Thank you for developing such a great R package! While using the momentuHMM package, I came across a small issue. On page 59 of the [Guide to using momentuHMM](https://cran.r-project.org/web/packages/momentuHMM/vignettes/momentuHMM.pdf), there is a formula that may have an issue.

In my opinion, the correct formula should be(logit is corrected as log):

$$
\begin{aligned}
& b_1=(14400-0) {\log}^{-1}\left(\beta_{l, 4}\right)+0 \\
& b_2=(14400-0) \log^{-1}\left(\beta_{l, 4}+\exp \left(\beta_{l, 5}\right)+0\right)+0 \\
& b_3=(14400-0) \log^{-1}\left(\beta_{l, 4}+\exp \left(\beta_{l, 5}\right)+0+\exp \left(\beta_{l, 6}\right)+0\right)+0 .
\end{aligned}
$$

Thank you again for developing such a great R package!

**Comment by bmcclintock** [comment, marked as answer]
Thanks for your message. {momentuHMM} uses the [logit link](https://en.wikipedia.org/wiki/Logit) for any parameters with finite lower and upper bounds. In this case, the lower bound is 0 and the upper bound is 14400, and the logit link ensures the real parameters stay within these bounds (as the inverse logit link is bound between 0 and 1). If one were to use the log link, then the lower bound would be 0 and the upper bound would infinite (as the inverse log link is bound between 0 and infinity).

**Comment by xjtusjtu** [comment]
Thank you for your response. I understand your point and the logic behind it. This is why the caption for table 2 states, "If user-specified parameter bounds are provided, then custom link functions are used instead of the defaults (see package documentation for further details)."

According to table 2, for the Weibull distribution, when the shape parameter is greater than 0, the default link function is "log." However, in this example, with finite bounds provided, the link function changes from "log" to "logit."

Once again, thank you for your response!
![image](https://github.com/bmcclintock/momentuHMM/assets/57665402/cbce9d32-f951-4348-b10f-1865338743e3)
